recipes/bgzip/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "bgzip" %}
+{% set version = "0.3.5dev0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/xialiu71/bgzip/archive/v{{ version }}.tar.gz
+  sha256: b671e03ca8508910f58ee620957dcbccf054164c4205518ef96005f57d37cc27
+
+build:
+  number: 0
+  skip: true  # [win]
+  binary_relocation: False  # [linux]
+  ignore_run_exports: 
+    - cython
+    - cpython
```

Comment:
I'm curious, why do you need those?

Reply:
Thanks for reviewing! Double checked and lines 16-19 aren't necessary, my bad. They've been removed. Removing 15 causes an error, should I add a comment in the file for that?

Reply:
I'm not 100% confidence with setting binary relocation to false there. My understanding is that this is rarely needed. @isuruf, do you mind taking a quick look?